adafruit_vs1053.py

```diff
@@ -204,14 +210,16 @@ def start_playback(self):
         # Set time to zero.
         self.decode_time = 0
 
-    def stop_playback(self):
+    def stop_playback(self) -> None:
         """Stop any playback of audio."""
         self._sci_write(
             _VS1053_REG_MODE,
             _VS1053_MODE_SM_LINE1 | _VS1053_MODE_SM_SDINEW | _VS1053_MODE_SM_CANCEL,
         )
 
-    def play_data(self, data_buffer, start=0, end=None):
+    def play_data(
+        self, data_buffer: bytearray, start: int = 0, end: Optional[int] = None
```

Comment:
`data_buffer` can be more general - based on use of `len()` and `SPI.write()`, it can be `ReadableBuffer` as imported from `circuitpython_typing`.